documentation/howto/os3_deployimage.adoc

```diff
@@ -0,0 +1,87 @@
+= Deploy a Docker Image to OpenShift 3
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+:toc:
+
+JBoss Tools' OpenShift 3 tooling allows users to deploy a Docker image to an OpenShift cluster from a Docker daemon.
+Deploying an image to an OpenShift cluster generates several resources to support the image which include:
+
+- ImageStream
+- Deployment Configuration
+- Service
+- Route 'Optional'
+
+*Note:* The following steps require:
```

Comment:
[NOTE]
====================================================
The following steps require:

- Connection to a Docker daemon already exist
- Connection to an OpenShift 3 cluster already exist
- Openshift 3 project exist
====================================================